docs/man.rst

```diff
@@ -95,13 +95,13 @@ The following list contains the available ``--extra-vars`` options.
 
   Controls whether to start the Quipucords server with supervisord. Contains a true or false value. Defaults to ``true``. Supply ``false`` to start the server without supervisord.
 
-``-e pkg_install_dir=~/packages``
+``-e tools_install_dir=~/volumes``
```

Comment:
why don't we call this `quipucords_home` or something?  This is not the tools install dir it's the directory used by quipucords server and possibly CLI.